htlcswitch/link_test.go

```diff
@@ -265,6 +265,9 @@ func TestChannelZeroAddLocalHeight(t *testing.T) {
 	// <---settle----
 	ctx.sendSettleBobToAlice(uint64(0), *inv.Terms.PaymentPreimage)
 
+	// Wait for Alice to pull the Settle off the mailbox queue.
+	<-time.After(5 * time.Second)
```

Comment:
This can be much lower, like in milli-seconds, no?

Reply:
Test needs to be moved to `lnwallet/channel_test.go` since it doesn't actually test any link-specific behavior.

Reply:
Test removed. It is more explicit which I like, so I could add it as a separate test in addition to expanding the one we have.

Reply:
Test moved to `lnwallet/channel_test.go` and is called `TestChannelZeroAddLocalHeight`